system_files/desktop/shared/usr/libexec/bazzite-bluetooth-sleep-workaround

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/bash
+
+devices=$(bluetoothctl devices Connected | awk '{print $2}')
+
+# shellcheck disable=SC2068
+for dev in $devices; do
+    [ -n "$dev" ] &&  bluetoothctl disconnect "$dev"
```

Comment:
Just adding this for reference if it will be needed.
when i had to turn off my windows VMs bluetooth adapter using a script at boot i had to do it something like
```bash
echo -e "select 00:E0:4C:77:99:88\npower off\n" | bluetoothctl
```
otherwise it would not reliably work.

not asking for a change, just adding it as a comment in case it ends up not activating reliably then this is something that can be tried :slightly_smiling_face:

Reply:
actually @Zeglius it will need some changes as this will fail on devices that would happen to have multiple bluetooth controllers.

see https://github.com/ublue-os/bazzite/pull/1416#issuecomment-2254628297

Reply:
Solved with commit b4c19487f1ed00e128997f91caae52e8ec769066